testsuite/unittests/test_symmetric_connections.sli

```diff
@@ -149,4 +151,91 @@ fail_or_die
   true exch { and } Fold
 } assert_or_die
 
+
+% The now following tests need the synapse model
+% gap_junction (which has require_symmetric=true) and
+% ergo the model hh_psc_alpha_gap, so
+% this test should only run if we have GSL
+statusdict/have_gsl :: not
+{
+  exit_test_gracefully
+}
+if
+
+/setup2
+{
+  ResetKernel
+  
+  /hh_psc_alpha_gap 4 Create
+  
+  /set1 1 2 cvgidcollection def
+  /set2 3 4 cvgidcollection def
+} def
+
+% Check that connections that require_symmetric cannot be created without symmetric=true
```

Comment:
Shouldn't there be one test checking that one-to-one Connect with gap_junction actually creates the expected connections? Or is that elsewhere? All further tests are just fail/pass, but do not check the result.